.gitignore

```diff
@@ -64,3 +64,4 @@ TestProject/Logs
 TestProject/Assets/Plugins
 TestProject/Assets/Samples*
 inst-dbg/
+package/com.unity.formats.usd/TestRunnerOptions.json.meta
```

Comment:
Why is this needed?
(any time .meta is git ignored, my spidy sense starts to tingle)

Reply:
This is a strange one: this file is a part of the package, but ignored by upm-ci pack. If I were to add a meta file, you'd fail validation because you'd have a meta without the corresponding file.